src/main/java/calculator/validator/AnotherLetterValidator.java

```diff
@@ -0,0 +1,20 @@
+package calculator.validator;
+
+public class AnotherLetterValidator {
+
+
+    public AnotherLetterValidator(String input) {
+
+        validateAnotherLetter(input);
+    }
+
+    private void validateAnotherLetter(String input) {
+        if (input.charAt(0) != '/' && (input.chars().anyMatch(c -> !Character.isDigit(c) && c != ',' && c != ':'))) {
+            throw new IllegalArgumentException("허용되지 않는 문자가 존재합니다.");
+        }
+        char Separator = input.charAt(2);
+        if (input.charAt(0) == '/' && input.substring(5).chars().anyMatch(c -> c != ',' && c != ':' && c != Separator && !Character.isDigit(c))) {
```

Comment:
조건에서 공통적인 부분은 묶어서 메서드로 빼도 괜찮을 것 같아요!
```suggestion
        if (input.charAt(0) == '/' && isInvalidCharacter(input.substring(5))) {
```